### DISCUSSION TITLE
Helix keymap extension for vscode

### DISCUSSION THREAD

**Discussion by cheparity** [helix-editor/helix#11702] [General]
I found a good enough extension simulating helix keybindings in vscode. Also can be found in vscode extension market.
[vscode-helix](https://github.com/jasonwilliams/vscode-helix)
Note that though it is still lacking some of helix features like Ctrl-C, Ctrl-S, etc., it is enough for most of my daily usage.

Enjoy! Maybe I would like to contribute to make it better.

**Comment by rahil627** [comment]
i use this! it's a great mix of helix in normal mode, vscode(/emacs?) in insert mode. However, no pop-up menus, so you've gotta be good at helix. ;) A few weird things often happen, but it still works well enough!
